pkg/option/config.go

```diff
@@ -2042,6 +2046,10 @@ type DaemonConfig struct {
 	// EnableK8sTerminatingEndpoint enables auto-detect of terminating state for
 	// Kubernetes service endpoints.
 	EnableK8sTerminatingEndpoint bool
+
+	// TCFilterPrio sets the priority of the cilium tc filter, enabling other
+	// filters to be inserted prior to the cilium filter.
+	TCFilterPrio int
```

Comment:
Can we keep the full word for the variable name, `TCFilterPriority`?

Reply:
Yeah, initially I was modeling it after the keyword used in the tc command. :) Then changed it to the full word for the user-facing option. I will fix this.